docs/src/database_import.md

```diff
@@ -1,94 +1,97 @@
-latter by the `monolith` type.
+- **Monolith approach** - The destination cluster is designed to host multiple
+  databases and different users, imported from the source cluster. This method
+  is available by way of the `monolith` type.
 
 !!! Warning
-    It is your responsibility to ensure that the destination cluster can
+    It's your responsibility to ensure that the destination cluster can
     access the source cluster with a superuser or a user having enough
-    privileges to take a logical backup with `pg_dump`. Please refer to the
-    [PostgreSQL documentation on "SQL Dump"](https://www.postgresql.org/docs/current/app-pgdump.html)
-    for further information.
+    privileges to take a logical backup using pg_dump. See the
+    [PostgreSQL documentation on pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html)
+    for more information.
 
 ## The `microservice` type
 
 With the microservice approach, you can specify a single database you want to
-import from the source cluster into the destination cluster. The operation is
-performed in 4 steps:
+import from the source cluster to the destination cluster. The operation is
+performed in four (optionally five) steps:
 
 - `initdb` bootstrap of the new cluster
-- export of the selected database (in `initdb.import.databases`) using
+- Export of the selected database (in `initdb.import.databases`) using
   `pg_dump -Fc`
-- import of the database using `pg_restore --no-acl --no-owner` into the
+- Import of the database using `pg_restore --no-acl --no-owner` into the
   `initdb.database` (application database) owned by the `initdb.owner` user
-- cleanup of the database dump file
-- optional execution of the user defined SQL queries in the application
-  database via the `postImportApplicationSQL` parameter
-- execution of `ANALYZE VERBOSE` on the imported database
+- Cleanup of the database dump file
+- Optional execution of the user-defined SQL queries in the application
+  database by way of the `postImportApplicationSQL` parameter
+- Execution of `ANALYZE VERBOSE` on the imported database
 
 ![Example of microservice import type](./images/microservice-import.png)
 
-For example, the YAML below creates a new 3 instance PostgreSQL cluster (latest
+For example, the YAML that follows:
+
+- Creates a three-instance PostgreSQL cluster (latest
 available major version at the time the operator was released) called
-`cluster-microservice` that imports the `angus` database from the
-`cluster-pg96` cluster (with the unsupported PostgreSQL 9.6), by connecting to
-the `postgres` database using the `postgres` user, via the password stored in
-the `cluster-pg96-superuser` secret.
+`cluster-microservice`
+- Imports the `angus` database from the
+`cluster-pg96` cluster (with the unsupported PostgreSQL 9.6) by connecting to
+the `postgres` database using the postgres user 
+- Uses the password stored in the `cluster-pg96-superuser` secret to connect to the `postges` database
 
```

Comment:
Please check this as I was trying to break up a long sentence and I wasn't 100% sure I retained the meaning.